R/.importConservation.R

```diff
@@ -0,0 +1,45 @@
+.importConservation <- function(fName) {
+    # Validate fName if file and stuff
+
+    # variable names subject to changes
+
+    # Setup
+    open_file(fName)
+    readline(n = 1)
+    currPos <- extract from read line
+    chromosomeNum <- extract from read line
+    # or could specify from function signature
+    outName <- "chromosome + chromosomenum.RDS"
+
+    # Whether to store coordinate with score
+    chromosomeScores <- vector or matrix size 20000
+
+    currSize <- size(chromosomeScores)
+
+    currIndex <- 1
+
+    while (length(entry <- readLines(opened_file, n = 1)) > 0 {
+        # decide on order later
+        # will there be empty data in between indices
+        # refer to email
+        if currIndex > currSize {
+            chromosomeScores
+            update currSize
```

Comment:
Actually you are increasing the number of rows by 10000 since the matrix has two columns. (Careful  :-)